### PR TITLE
#4557 Added null if there is no alt text to image

### PR DESCRIPTION
**Pull request by thinkbulecount2**
@NateWr [see issue here](https://github.com/pkp/pkp-lib/issues/4557)

### CODE REVIEW

File: templates/common/footer.tpl [hunk 1]
```diff
@@ -14,7 +14,7 @@
 <div class="pkp_structure_footer" role="contentinfo">
 	<div class="pkp_brand_footer">
 		<a href="{$pkpLink}">
-			<img alt="{translate key="common.publicKnowledgeProject"}" src="{$baseUrl}/{$brandImage}">
+			<img alt="{translate key="common.publicKnowledgeProject"|default:'null'}" src="{$baseUrl}/{$brandImage}">
```
Comment (NateWr) [inline templates/common/footer.tpl]: You can leave this change out.

When you see a `{translate key="..."}` used, that comes directly from the locale strings. For that reason, you can be sure that the alt tag will never be empty.

Comment (thinkbulecount2) [reply templates/common/footer.tpl]: Ok! So I roll back my changes?

Comment (NateWr) [reply templates/common/footer.tpl]: Yep, so you can leave lib/pkp out of the changes altogether

**Comment by NateWr** [comment]
Just closing this since it's no longer being used.

**Pull request closed without merging**